titus_isolate/isolate/workload_manager.py

```diff
@@ -39,6 +42,11 @@ def __init__(self, cpu, cgroup_manager, static_cpu_allocator):
         log.info("Created workload manager")
 
     def add_workload(self, workload):
+        if self.__occupies_entire_cpu(workload, self.__cpu):
```

Comment:
Does it make sense to put this behind an FP / configurable?

Reply:
Yes.  I'll do that.